build.gradle

```diff
@@ -1,26 +1,30 @@
-apply plugin: 'java'
-apply plugin: 'groovy'
-apply plugin: 'maven'
-apply plugin: 'maven-publish'
-apply plugin: 'idea'
-
-group = 'net.fabric.gradle'
-version = '0.0.1'
-sourceCompatibility = targetCompatibility = '1.8'
-
-if (System.getenv().BUILD_NUMBER) {
-    version = version + "." + "${System.getenv().BUILD_NUMBER}"
+plugins {
+    id 'java'
+    id 'groovy'
+    id 'maven'
+
+    id 'idea'
+    id 'eclipse'
+
+    id 'net.minecrell.licenser' version '0.2.1'
 }
 
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+group = 'net.fabric'
+archivesBaseName = project.name.toLowerCase()
+version = '0.0.1-SNAPSHOT'
```

Comment:
http://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it